arrows/ffmpeg/ffmpeg_video_input.cxx

```diff
@@ -285,7 +285,6 @@ class ffmpeg_video_input::priv
         this->f_start_time = -1;
         return false;
     }
-
```

Comment:
Not critical, but I would undo the whitespace changes in this function.  They aren't relevant to this branch and don't make any more compliant with style guidelines